### PR TITLE
Update whispers to 2.1.0

### PR DESCRIPTION
**Pull request by adeptex**
Hey there, I am the creator of whispers. I saw that awesomely it was included into BlackArch, and I thought I would contribute by updating it to the latest version. [Ownership changed](https://github.com/adeptex/whispers/blob/2.0.6/RELEASE_NOTES.md#dizzy-licensing-changes-dizzy) from Skyscanner to my personal account. In addition, Skyscanner's [version 1](https://github.com/Skyscanner/whispers/) is no longer maintained.

Let me know if you need any additional details. Thanks!

### CODE REVIEW

File: packages/whispers/PKGBUILD [hunk 1]
```diff
@@ -2,17 +2,17 @@
 # See COPYING for license details.
 
 pkgname=whispers
-pkgver=1.5.3.r13.g5832bf7
+pkgver=2.1.0.r01
```
Comment (noraj) [inline packages/whispers/PKGBUILD:5]:
```
$ git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
2.1.0.r0.g8cdc6b1
```

File: packages/whispers/PKGBUILD [hunk 2]
```diff
@@ -2,17 +2,17 @@
...
 pkgrel=2
-pkgdesc='Identify hardcoded secrets and dangerous behaviours.'
+pkgdesc='Identify hardcoded secrets in static structured text.'
 arch=('any')
 groups=('blackarch' 'blackarch-code-audit')
-url='https://github.com/Skyscanner/whispers/'
-license=('Apache')
+url='https://github.com/adeptex/whispers/'
+license=('GPL-3.0')
```
Comment (noraj) [inline packages/whispers/PKGBUILD:11]: change to `GPL3

its has to respect `/usr/share/licenses/common/` names as mentioned here https://wiki.archlinux.org/title/PKGBUILD#license

**Review by noraj — CHANGES_REQUESTED** [review]

**Comment by noraj** [comment]
> Let me know if you need any additional details. Thanks!

Thanks for your contribution, could you apply the requested changes please?

**Comment by adeptex** [comment]
hey @noraj, done and done! thanks

**Review by noraj — APPROVED** [review]

**Comment by noptrix** [comment]
@adeptex thanks a lot

**Pull request merged**